src/components/node/banner/index.tsx

```diff
@@ -1,7 +1,31 @@
+import { useRef, useEffect, useState } from 'react'
 import { VaBanner } from '@department-of-veterans-affairs/component-library/dist/react-bindings'
 import { isEmpty } from 'lodash'
+import { recordEvent } from '@/utils/recordEvent'
+import { NodeMetaInfo, NodeResourceType } from '@/types/node'
 
 const BannerComponent = ({ node }): JSX.Element => {
+  const [isClicked, setIsClicked] = useState(false)
```

Comment:
This logic doesn't need to be in the banner component

Reply:
Whoops that was in there from earlier, removed